packages/shared_preferences/shared_preferences_linux/example/pubspec.yaml

```diff
@@ -5,14 +5,12 @@ dependencies:
   flutter:
     sdk: flutter
   shared_preferences: any
-  shared_preferences_linux: ^0.1.0
 
 dependency_overrides:
   shared_preferences_linux:
     path: ../
-  # Remove this override once the endorsement is published.
-  shared_preferences:
-    path: ../../shared_preferences/
+  shared_preferences_platform_interface:
+    path: ../../shared_preferences_platform_interface
```

Comment:
Do we actually need this to be an override, or can we use a published version? The latter would be preferable.

Reply:
We have to wait for shared_preferences to bu published and depend on the null-safety versions of `shared_preferences_linux` before. We can then remove it yeah.

Reply:
We can't migrate and publish the app-facing package before the implementations. (See my other comment about removing the circular dependency.)